src/generator/operations.ts

```diff
@@ -204,10 +204,10 @@ function formatParamValue(param: Parameter, imports: ImportManager): string {
 }
 
 // use this to generate the code that will help process values returned in response headers
-function formatHeaderResponseValue(header: string, schema: Schema, imports: ImportManager, respObj: string): string {
+function formatHeaderResponseValue(propName: string, header: string, schema: Schema, imports: ImportManager, respObj: string): string {
```

Comment:
What's the difference between propName and header?

Reply:
`propName` is the name of the property as it appears in the struct.  `header` is the actual header, e.g. `x-ms-foo`

Reply:
Ah I see. Thanks for the explanation!